spec/presenters/presenter_spec.rb

```diff
@@ -0,0 +1,34 @@
+RSpec.describe Wpcc::Presenter do
+  subject do
+    described_class.new(object, view_context: view_context)
+  end
+
+  let(:object) { double(:object) }
+  let(:view_context) { double(:view_context) }
```

Comment:
if you add `ActionController::Base.new.view_context` then you don't need to set stubs and mocks for the translation that you do above.